notebooks/tabular_examples/model_agnostic/Simple California Demo.ipynb

```diff
@@ -0,0 +1,293 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Simple California Demo\n",
+    "\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This notebook shows how to use build a hierarchical clustering of the input features and use it to explain a single instance. This is a good way to explain a single instance when the number of input features is large.\n",
+    "When given a balanced partition tree PartitionExplainer has $O(M^2)$ runtime, where $M$ is the number of input features. This is much better than the $O(2^M)$ runtime of KernelExplainer."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import scipy as sp\n",
+    "import scipy.cluster\n",
+    "from xgboost import XGBRegressor\n",
+    "\n",
+    "import shap\n",
+    "\n",
+    "seed = 2023\n",
+    "np.random.seed(seed)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "end_time": "2023-10-14T10:50:49.957930069Z",
+     "start_time": "2023-10-14T10:50:49.957130605Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Train a model"
```

Comment:
We need to make sure that the section headings are not the same as the title, otherwise this happens in the rendered output:

![image](https://github.com/shap/shap/assets/30731072/a9de7f78-7443-4e17-8d5b-eecd504c922c)

So this needs to be `## Train a model` and same for subsequent headings